client/components/costume/SingleCostume.js

```diff
@@ -21,6 +22,12 @@ class SingleCostume extends Component {
     this.setState({ quantity: ev.target.value });
   }
 
+  addCostumeToCart(costumeId, quantity) {
+    this.props.addCostumeToCart(costumeId, quantity);
+    this.props.fetchCart();
```

Comment:
I think we need await on lines 26 and 27, before going to /cart

Reply:
async / await added